src/controllers/DeliveryEmployeeController.ts

```diff
@@ -0,0 +1,49 @@
+import { Request, Response } from "express";
+import { createDeliveryEmployee, deleteDeliveryEmployee, editDeliveryEmployee, getDeliveryEmployee, getDeliveryEmployees } from "../services/DeliveryEmployeeService";
+
+export const getDeliveryEmployeeList = async (req: Request, res: Response): Promise<void> => {
```

Comment:
can we use this format for all other functions please. `(req: Request, res: Response)` instead of `(req: express.Request, res: express.Response)`

Reply:
thanks Declan ,this is complete